README.rst

```diff
@@ -123,11 +126,36 @@ The example produces a file greeting all names included in the
      Hello Jane Doe!
      Hello Joe Bloggs!
 
-Running the example on REANA cloud
+Running the above example on REANA
 ==================================
 
-We are now ready to run this example and on the `REANA <http://www.reana.io/>`_
-cloud.
+We are now ready to run this example and on the `REANA <http://www.reana.io/>`_ platform.
+
+There are two ways to execute this analysis example on REANA.
+
+If you would like to simply launch this analysis example on the REANA instance
+at CERN and inspect its results using the web interface, please click on one of
+the following badge:
```

Comment:
... one of the following badges, depending on which workflow system (CWL, Serial, Snakemake, Yadage) you would like to use: